src/averageAges.js

```diff
@@ -38,6 +49,18 @@ function calculateMenAverageAge(people, century) {
  */
 function calculateWomenAverageAge(people, withChildren) {
   // write code here
+  const womens = people.filter(({ sex }) => sex === 'f');
```

Comment:
That's actually interesting one, in English they don't have such word as womens. Plural for women will actually be women (pronounced like wimin). And 'mens' variable is also a bit wrong, it should be men. I'd suggest either remove [s] ending in both of these variables or rename them to females/males.

Reply:
You're right! Thanks! I need to visit English classes more 😄

Reply:
It's actually kinda hard to find free class to visit. All of them are filled up when i check 😄